src/main/java/com/codegym/jrugotom5/controller/AdvertController.java

```diff
@@ -27,4 +29,10 @@ public List<AdvertBasicInfoDTO> getAllAdverts() {
     public List<AdvertFullInfoDTO> getAdverts(@RequestParam LocalDate from, LocalDate to) {
         return advertService.getAdvertsByDateRange(from, to);
     }
+
+    @PostMapping
+    public ResponseEntity<AdvertFullInfoDTO> createAdvert(@RequestBody @Valid AdvertCreateDTO advertCreateDTO) {
+        AdvertFullInfoDTO advertFullInfoDTO = advertService.createAdvert(advertCreateDTO);
+        return new ResponseEntity<>(advertFullInfoDTO, HttpStatus.CREATED);
```

Comment:
I suppose it's better to use Method annotation (it's more clear) instead of "ResponceEntity"